Modules/binascii.c

```diff
@@ -334,14 +334,15 @@ binascii_a2b_uu_impl(PyObject *module, Py_buffer *data)
 binascii.b2a_uu
 
     data: Py_buffer
-    /
```

Comment:
It looks to me that change in bpo-25357 was not intentional. Perhaps we can revert it. Open a new issue for this.